docs/src/overrides/home.html

```diff
@@ -3,311 +3,311 @@
+                </header>
+                <p class="md-typeset">The potential of Earth Observation data is immense in helping us understand our
+                    ever-changing planet. To fulfill this potential, we must provide EO data that can be easily
+                    integrated into modern applications and decision-making tools. It&apos;s important that it&apos;s easy to
+                    discover, interoperable, ingestible, and optimized for applications. This is a significant challenge
+                    for many organizations, and our mission at Development Seed is to address this challenge.</p>
+            </div>
+        </div>
+    </section>
+
+    <section class="md-content dark-section" data-md-color-scheme="default">
+        <div class="md-grid" data-md-component="content">
+            <div class="md-content__inner">
+                <header class="md-typeset text-centered">
+                    <h5 id="eoapi-in-action" class="supheading">eoAPI in action</h5>
+                </header>
+                <div class="figure-collection" id="eoAPI-examples">
+                    <figure>
+                        <figcaption class="md-typeset">
+                            <h2>NASA IMPACT&apos;s VEDA Project</h2>
+                            <p>NASA IMPACT is leading the way for how NASA (and space agencies worldwide) can increase
+                                the
+                                societal impact of their data through cloud-optimized data distribution. They&apos;ve
+                                invested
+                                significant resources in advancing modern public EO data standards, tools, and
+                                platforms.
+                                You
+                                can see eoAPI services in action throughout the <a href="https://earthdata.nasa.gov/dashboard" rel="noopener" target="_blank">VEDA platform</a>.</p>
+                        </figcaption>
+                        <img src="../img/eoapi_veda.png" alt="NASA VEDA platform screenshot" loading="lazy">
+                    </figure>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="md-content secondary-section" data-md-color-scheme="default">
+        <div class="md-grid" data-md-component="content">
+            <div class="md-content__inner">
+                <header class="md-typeset text-centered">
+                    <h5 id="trusted-by" class="supheading">Trusted By</h5>
+                </header>
+                <div class="feature-grid logo-grid" style="--cols: 3;">
+                    <figure>
+                        <img src="../img/nasa_logo.svg" alt="NASA logo" loading="lazy" height="88px">
+                    </figure>
+                    <figure>
+                        <img src="../img/msft_logo.svg" alt="Microsoft Logo" loading="lazy" height="64px">
+                    </figure>
```

Comment:
We should remove Microsoft!!! They do use a similar approach (will the base services) but it's not eoapi!

Reply:
Removed.